src/v/pandaproxy/handlers.cc

```diff
@@ -303,4 +307,24 @@ get_consumer_offsets(server::request_t rq, server::reply_t rp) {
       });
 }
 
+ss::future<server::reply_t>
+post_consumer_offsets(server::request_t rq, server::reply_t rp) {
+    auto group_id = kafka::group_id(rq.req->param["group_name"]);
+    auto member_id = kafka::member_id(rq.req->param["instance"]);
+
+    // If the request is empty, commit all offsets
+    auto req_data = rq.req->content.length() == 0
+                      ? std::vector<kafka::offset_commit_request_topic>()
+                      : ppj::partition_offsets_request_to_offset_commit_request(
+                        ppj::rjson_parse(
+                          rq.req->content.data(),
+                          ppj::partition_offsets_request_handler()));
+
+    return rq.ctx.client
+      .consumer_offset_commit(group_id, member_id, std::move(req_data))
+      .then([rp{std::move(rp)}](kafka::offset_commit_response) mutable {
```

Comment:
does it matter if the commit offset response contains an error?

Reply:
> does it matter if the commit offset response contains an error?

It doesn't. But a `consumer_error` can be thrown, and that does. Thanks.

I strongly suspect that [the documentation](https://docs.confluent.io/platform/6.0.2/kafka-rest/api.html#post--consumers-(string-group_name)-instances-(string-instance)-offsets) is incomplete, dripping the result feels wrong.